### DISCUSSION TITLE
Deployment fails with NOW_SANDBOX_WORKER_MAX_LAMBDA_SIZE for a 404 on remix app build

### DISCUSSION THREAD

**Discussion by jacargentina** [vercel/community#2187] [Help]
### Summary

Error: The Serverless Function "404" is 50.07mb which exceeds the maximum size limit of 50mb. Learn More: https://vercel.link/serverless-function-size
--
Deployment completed
NOW_SANDBOX_WORKER_MAX_LAMBDA_SIZE: The Serverless Function "404" is 50.07mb which exceeds the maximum size limit of 50mb.

### Additional information

I don't have any "404" route I can handle, I don't know how to solve this error, any hand?

**Comment by swarnava** [comment]
Hi,

Do you have a repository which we can use to reproduce?

**Reply by jacargentina** [reply]
https://github.com/jacargentina/iresucito.git
